src/sql/workbench/contrib/notebook/test/browser/cellToolbarActions.test.ts

```diff
@@ -218,5 +218,5 @@ export async function createandLoadNotebookModel(codeContent?: nb.INotebookConte
 		layoutChanged: undefined,
 		capabilitiesService: undefined
 	};
-	return new NotebookModel(defaultModelOptions, undefined, undefined, undefined, undefined, undefined, undefined);
+	return new NotebookModel(defaultModelOptions, undefined, undefined, undefined, undefined, undefined, undefined, undefined);
```

Comment:
Do we need to pass in undefined here for the optional parameter?

Reply:
Fixed